website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,288 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another Windows Subsystem for Linux (WSL) distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, [Podman Desktop configures Podman in a dedicated Windows Subsystem for Linux (WSL) distribution: the Podman Machine](/docs/podman/creating-a-podman-machine.md).
+It also configures the Windows Podman client to communicate with the Podman Machine.
+However, it does not configure your other WSL distributions.
+
+You might have other WSL distributions running, and want to access from there to your Podman Desktop containers.
+
+This tutorial focuses on the most common context to walk you through the steps to configure your WSL distribution:
+
+- Ubuntu distribution of Linux.
+- Default Podman Machine.
+
+In foldable details, you can find alternative steps for least common contexts:
+
+- Custom WSL distribution.
+- Custom Podman Machine.
+
+## Configuring your WSL distribution
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. To communicate with the remote Podman Machine, you need a Podman client.
+
+   Install the Podman client:
+
+   ```shell-session
+   $ sudo apt update
+   $ sudo apt install podman
+   ```
+
+   The package might set up local Podman resources.
+   However, you plan not to use them.
+
+   :::tip
+
+   When the Podman client is already installed, you might skip this step.
+
+   :::
+
+   :::info
+
+   On a custom WSL distribution, the installation step might be different.
+   See [Installing Podman on Linux](https://podman.io/docs/installation#installing-on-linux).
+
+   :::
+
+1. The communication channel between your WSL distribution and the Podman Machine is a special file (a socket).
+   The Podman Machine creates this file with specific permissions.
+   On your WSL distribution, to communicate with the Podman Machine, your user must have the write permissions on this file.
+
+   To give access to the remote Podman machine to your user, assign group membership:
+
+   ```shell-session
+   $ sudo usermod --append --groups 10 your_user
+   ```
+
+1. Exit your session on the WSL distribution to apply the new group membership:
+
+   ```shell-session
+   $ exit
+   ```
+
+1. Start a new WSL session:
+
+   ```shell-session
+   > wsl
+   ```
+
```

Comment:
I will move these steps at the end to:

- Have one step less.
- Avoid one exit/reconnect on WSL.

Reply:
```
$ sudo groupadd --gid 10 --force podman
```